Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -70,674 +74,668 @@ namespace Unity.MeshSync
 /// </summary>
 internal delegate void DeleteInstanceHandler(string path);
```

Comment:
separate PR for this reformat here: https://github.com/unity3d-jp/MeshSync/pull/701